commodore/component/template.py

```diff
@@ -13,6 +14,9 @@
 from commodore.helpers import yaml_load, yaml_dump
 
 
+slug_regex = re.compile("^[a-z][a-z0-9-_]+[a-z0-9]$")
```

Comment:
Should we even allow the `_` (underscore) in names?
It might be really confusing if `-` gets converted to `_`, you don't know anymore if the `_` was being converted or not...

Reply:
yeah, I was thinking about this as well, I'm fine with disallowing `_`.